### PR TITLE
Add Unity Shirts and Kilusha Items to Items global

### PR DESCRIPTION
**Pull request by Era-Lusiphur**
Setting these up for future PRs to avoid global conflicts if they linger for a while.

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [o] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Comment by Era-Lusiphur** [comment]
This marks the first addition of handling a +1 item in the items global that I could see. Had a quick discussion with @zach2good about best way to handle it, but want @claywar 's OK to use this format of _P1, _P2, etc. going forward, since this has been his baby so far.

File: scripts/globals/items.lua [hunk 1]
```diff
@@ -342,4 +368,15 @@ xi.items =
...
+    PIEUJE_UNITY_SHIRT              = 25734,
+    AYAME_UNITY_SHIRT               = 25735,
+    IVINCIBLE_SHIELD_UNITY_SHIRT    = 25736,
+    APURURU_UNITY_SHIRT             = 25737,
+    MAAT_UNITY_SHIRT                = 25738,
+    ALDO_UNITY_SHIRT                = 25739,
+    JAKOH_WAHCONDALO_UNITY_SHIRT    = 25740,
+    NAJA_SALAHEEM_UNITY_SHIRT       = 25741,
+    FLAVIRIA_UNITY_SHIRT            = 25742,
+    YORAN-ORAN_UNITY_SHIRT          = 25743,
```
Comment (zach2good) [inline scripts/globals/items.lua]: Contrary to how we deal with filenames, I think in lua scripting we replace hypens with underscores

File: scripts/globals/items.lua [hunk 2]
```diff
@@ -290,6 +315,7 @@ xi.items =
     KODACHI_OF_TRIALS               = 17773,
     MUMEITO                         = 17809,
     TACHI_OF_TRIALS                 = 17815,
+    ANIMATOR_P1                     = 17857,
```
Comment (zach2good) [inline scripts/globals/items.lua:318]: I like _P1, _P2 as a pattern

Comment (TeoTwawki) [reply scripts/globals/items.lua:318]: P for "Plus" works as long as we don't hit anything that actually has that as part of its text string. If that happens someday we can just put the whole word.

Comment (TeoTwawki) [reply scripts/globals/items.lua:318]: we may eventually need "minus" representation also. Some items are named that way but atm none would need handled here.

Comment (Xaver-DaRed) [reply scripts/globals/items.lua:318]: I would go with PLUS_1, mainly becouse it isnt much larger and I dont imediately read it as "Player one" but...

**Review by claywar — APPROVED** [review]

**Pull request merged**